packages/react-reconciler/src/ReactFiberClassComponent.js

```diff
@@ -151,10 +151,18 @@ export function applyDerivedStateFromProps(
 
 export default function(
   legacyContext: LegacyContext,
-  scheduleWork: (fiber: Fiber, expirationTime: ExpirationTime) => void,
-  computeExpirationForFiber: (fiber: Fiber) => ExpirationTime,
+  scheduleWork: (
+    fiber: Fiber,
+    startTime: ExpirationTime,
+    expirationTime: ExpirationTime,
+  ) => void,
+  computeExpirationForFiber: (
+    currentTime: ExpirationTime,
```

Comment:
Above we have `startTime` as the first argument in the `computeExpirationForFiber` type, and here we have `currentTime` as the first argument. Is that intentional?